src/transformers/models/speech_encoder_decoder/modeling_speech_encoder_decoder.py

```diff
@@ -260,6 +266,13 @@ def get_output_embeddings(self):
     def set_output_embeddings(self, new_embeddings):
         return self.decoder.set_output_embeddings(new_embeddings)
 
+    def freeze_feature_extractor(self):
+        """
+        Calling this function will disable the gradient computation for the feature extractor of the speech encoder so
+        that its parameters will not be updated during training.
+        """
+        self.encoder.freeze_feature_extractor()
```

Comment:
Like I said, not super fan of the name as it could be confused with the feature extractor that preprocesses the data.